beeline-spring-boot-starter/src/test/java/io/honeycomb/beeline/spring/mockmvctests/MockMvcTest.java

```diff
@@ -549,24 +549,6 @@ public void WHEN_makingARequestToEndpointWithAOPAnnotation_EXPECT_aopSpanToBeSub
         assertThat(requestSpanFields).containsEntry("response.status_code", 200);
     }
 
-    @Test
-    public void WHEN_makingARequestWithHoneycombHeaderSpecifyingADataset_EXPECT_AllEventsToSendToDataset() throws Exception {
-        final PropagationContext context = new PropagationContext("current-trace-1", "parent-span-1", "myDataset", Collections.singletonMap("trace-field", "abc"));
-        final Map<String, String> headers = Propagation.honeycombHeaderV1().encode(context).get();
-
-        final MockHttpServletRequestBuilder builder = get("/annotation-span")
-            .header("x-application-header", "fish");
-        headers.forEach((k,v) -> builder.header(k, v));
-        mvc.perform(builder);
-
-        final List<ResolvedEvent> eventFields = captureNoOfEvents(2);
-        final ResolvedEvent aopSpan = eventFields.get(0);
-        final ResolvedEvent requestSpan = eventFields.get(1);
-
-        assertThat(aopSpan.getDataset()).isEqualTo("myDataset");
```

Comment:
Instead of removing this test that asserts the dataset is propagated, could we instead assert that the dataset does NOT get propagated? e.g. 

```java
assertThat(aopSpan.getDataset()).isEqualTo("testDataset");
assertThat(requestSpan.getDataset()).isEqualTo("testDataset");
```